File: 태성/0621/이상한문자만들기.js
const s = "try   hello world"	

function solution(s) {
    var answer = '';
    var ss = s.split(' ')
    console.log(ss)
    for (let sss of ss) {
        var pre = ''
        for (let i in sss) {
            
            if (i % 2 === 0) {
                pre += sss[i].toUpperCase()
            } else {
                pre += sss[i].toLowerCase()
            }
        }
        answer += pre
        answer += ' '
    }
    answer = answer.substring(0, answer.length-1)
    return answer;
}

console.log(solution(s))

File: 태성/LV1/0621/이상한 문자 만들기.py
def solution(s):
    answer = ''
    n = 0
    for ss in s:
        if ss == ' ':
            answer += ss
            n = 0
        else:
            if n%2 == 0:
                answer += ss.upper()
                n += 1
            else:
                answer += ss.lower()
                n += 1
    return answer